src/contentmethods/methods.py

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
I am not sure contentmethods and methods are super useful names...... what are these about? give a name that hints at  what they do, not what they are.

Reply:
Actually want to discuss about it. Because SWH api docs have api's nearly in 3-4 for subtopics like about content, about origin, about ops(release, builds etc) then some miscllaneous topics. So i thought we'll make seorate folders for all these seprate topics and will keep their related methods/api in one file inside those folders. So want discuss how to go about it.

Reply:
Actuall tushar suggested me to just keep one file in src folder so i did that way and new file name is swh_api_functions.py